app/coffee/DocManager.coffee

```diff
@@ -30,9 +30,9 @@ module.exports = DocManager =
 				else
 					callback err, doc
 
-	getAllDocs: (project_id, callback = (error, docs) ->) ->
+	getAllNonDeletedDocs: (project_id, callback = (error, docs) ->) ->
 		DocArchive.unArchiveAllDocs project_id, (error) ->
```

Comment:
Do we want to only unarchive non deleted docs here?  Different endpoint again in DocArchivemanager.